submission/submission-security/src/main/kotlin/ebi/ac/uk/security/integration/SecurityEvents.kt

```diff
@@ -0,0 +1,20 @@
+package ebi.ac.uk.security.integration
+
+import ebi.ac.uk.security.integration.model.events.UserPreRegister
+import ebi.ac.uk.security.integration.model.events.UserRegister
+import io.reactivex.Observable
+import io.reactivex.subjects.PublishSubject
+
+class SecurityEvents {
+
+    companion object {
+
+        // Subjects
+        internal val userPreRegister by lazy { PublishSubject.create<UserPreRegister>() }
+        internal val userRegister by lazy { PublishSubject.create<UserRegister>() }
+
+        // Observables
+        fun userPreRegister(): Observable<UserPreRegister> = userPreRegister
```

Comment:
What is pre-register? I'm assuming before activation? If so, probably we should use other name

Reply:
Yes before activation, any suggestion? initially, I called them `activation` and `registration` but that was even easier to confuse for me.